web/src/main/webapp/xslt/ui-metadata/form-builder.xsl

```diff
@@ -807,7 +807,34 @@
 
                     If only one choice, make a simple button
               -->
+              <xsl:when test="$qualifiedName = 'gfc:code' and $schema='iso19110'">
```

Comment:
Changes in iso19110 schema are related to this feature? I think no, but probably were committed in the same repository for other feature, please check and if not relevant would be good to remove and send them in other pull request.